Jenkinsfile

```diff
@@ -730,7 +730,7 @@ def frontendBuilderTemplateUpdated() {
 }
 
 def frontendUpdated() {
```

Comment:
Are these fields necessary if they're just returning true?

Reply:
I may re-instate that functionality.  no idea why the grep pattern started failing...

Reply:
kk